chrome/application-services-navigation.json

```diff
@@ -9,20 +9,51 @@
             "href": "/application-services/overview"
         },
         {
-            "title": "Streams for Apache Kafka",
+            "appId": "applicationServices",
+            "title": "API Management",
+            "href": "/application-services/api-management"
+        },
+        {
+            "appId": "applicationServices",
+            "title": "Connectors",
+            "href": "/application-services/cos"
+        },
+        {
+            "appId": "applicationServices",
+            "title": "Data Science",
+            "href": "/application-services/data-science"
+        },
+        {
+            "appId": "applicationServices",
+            "title": "Service Accounts",
+            "href": "/application-services/service-accounts"
+        },
+        {
+            "title": "Service Registry",
             "expandable": true,
             "routes": [
                 {
                     "appId": "applicationServices",
-                    "title": "Kafka Instances",
+                    "title": "Service Registry Instances",
                     "isBeta": true,
-                    "href": "/application-services/streams/kafkas"
-                },
+                    "href": "/application-services/service-registry"
+                },              
+                {
+                    "title": "Documentation",
+                    "isExternal": true,
+                    "href": "https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka"
```

Comment:
This is the URL for Kafka documentation. There's a separate URL for Service Registry documentation:

https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry

Reply:
Thanks :), I have updated the URL.